turbo.json

```diff
@@ -3,50 +3,50 @@
   "baseBranch": "origin/master",
   "pipeline": {
     "build": {
-      "cache": false,
+      "cache": true,
```

Comment:
I think these default to `true`? Could these all be removed then?

Reply:
It's more for visibility to be honest, just so it's really obvious where we're utilising it 👍🏼